contrib/slack.php

```diff
@@ -125,6 +125,7 @@ function checkSlackAnswer($result)
         'title' => get('slack_title'),
         'text' => get('slack_success_text'),
         'color' => get('slack_success_color'),
+        'fields' => get('slack_fields'),
```

Comment:
We should define it before using. Or it will fail for used without slack_fields.
Add 
```php
/* Some examples
 * ```php
 * ...
 * ```
 */
set('slack_fields', []);
```

Reply:
Wouldn't this be nicer;

```
'fields' => get('slack_fields') ?? [],
```

?

Reply:
This will throw an exception, so get(…, []);

But better to dectare separately as we can add comment with an example and it will be added to docs by bin/docgen

Reply:
Ok done